app/models/user.rb

```diff
@@ -1,14 +1,40 @@
 class User < ApplicationRecord
-  
-  attr_accessor :remember_token
-  before_save {email.downcase!}
-  validates :name, presence: true, length: {maximum: Settings.validates.name.maximum}
+  attr_accessor :remember_token, :activation_token
+  before_save   :downcase_email
+  before_create :create_activation_digest
+
+  validates :name, presence: true, length:
+    {maximum: Settings.validates.name.maximum}
   VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
-  validates :email, presence: true, length: {maximum: Settings.validates.email.maximum},
-    format: {with: VALID_EMAIL_REGEX},uniqueness: {case_sensitive: false}
+  validates :email, presence: true, length:
+    {maximum: Settings.validates.email.maximum}, format:
+      {with: VALID_EMAIL_REGEX},uniqueness: {case_sensitive: false}
```

Comment:
As you already used 2 space right alignment in the previous line, now for this line you don't need 2 more space here. Just align it with it's previous line